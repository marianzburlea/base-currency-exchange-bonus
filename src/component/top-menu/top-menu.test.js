import React from "react";
import renderer from "react-test-renderer";
import TopMenu from ".";

describe("@TopMenu Component", () => {
  it("should match the snapshot", () => {
    const tree = renderer.create(<TopMenu />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
