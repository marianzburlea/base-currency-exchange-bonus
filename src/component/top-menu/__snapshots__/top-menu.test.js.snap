// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@TopMenu Component should match the snapshot 1`] = `
<nav
  className="sc-bdVaJa gUwoOw"
>
  <a
    aria-current="page"
    href="/"
    onClick={[Function]}
  >
    Home Page
  </a>
  <a
    href="/about"
    onClick={[Function]}
  >
    About
  </a>
  <a
    href="/exchange"
    onClick={[Function]}
  >
    Exchange Rate
  </a>
</nav>
`;
